qcodes_contrib_drivers/drivers/Keysight/SD_common/SD_AWG.py

```diff
@@ -283,6 +283,35 @@ def set_trigger_io(self, value: int, verbose: bool = False) -> Any:
         result = self.awg.triggerIOwrite(value)
         value_name = f'set io trigger output to {value}'
         return result_parser(result, value_name, verbose)
+    
+    def set_marker_config(self, channel_number: int, markerMode: int,
+                          trgPXImask: int, trgIOmask: int, markerValue: int, syncMode: int ,length: int,
+                          delay:int, verbose: bool) -> Any:
+        """
+        Configures the marker output from the trigger port
+
+        Args:
+            channel_number : trigger channel number
+            markerMode : 0 disabled, 1 on start event, 2 On first sample of waveform (after starting delay), 3 on every cycle
+            trgPXImask : Mask to select PXI triggers to use
+            trgIOmask : Mask to select front panel triggers to use
+            markerValue : 0 = marker signal is default low, pulses high. 1 = marker signal is default high, pulses low
+            syncMode : 0 = syncronized to CLKsys. 1 = syncronized to 10MHz clock
+            length : Marker pulse length (length * clock period * 5)
+            delay : Delay to add before pulse (delay * clock period * 5)
+            verbose: boolean indicating verbose mode
+        """
+        result = self.awg.AWGqueueMarkerConfig(
+            nAWG=channel_number,
+            markerMode=markerMode,
+            trgPXImask=trgPXImask,
+            trgIOmask=trgIOmask,
+            value=markerValue,
+            syncMode=syncMode,
+            length=length,
+            delay=delay
+        )
+        return result_parser(result, verbose)
```

Comment:
I think you forgot to correctly use `result_parser ` function. It requires 3 args in the function construct, which the second arg. is an string intended to pass the name of what you want to parse:
`return result_parser(result, "pass here whatever name you like", verbose)`

Reply:
Thanks for this message! I misunderstood what result parser was doing. I submitted a new piece of code which has removed it. This code is working on my hardware in the lab.

Reply:
Sure.
So, verbose is not applicabale anymore with this change.

Reply:
removed verbose argument